ts/Dashboards/EditMode/ConfirmationPopup.ts

```diff
@@ -64,6 +64,11 @@ class ConfirmationPopup extends BaseForm {
         editMode: EditMode,
         options?: ConfirmationPopup.Options
     ) {
+        iconsURL =
+            options && options.close && options.close.icon ?
+                options.close.icon :
+                iconsURL;
```

Comment:
`options?.close?.icon || iconsURL;` should be more transparent

Reply:
But increases transpiled code. We looking into proper support for optionals, but at the moment we should not actively change conditions imho.